climakitae/core/data_interface.py

```diff
@@ -991,7 +995,24 @@ def _update_scenarios(self):
             if scen in historical_scenarios
         ]
         self.param["scenario_historical"].objects = scenario_historical_options
-        if self.scenario_historical not in scenario_historical_options:
+
+        # test to check if one list is contained within another as python doesn't have that out of the box
```

Comment:
Maybe specify `check if A is contained in B...` for clarity of checking which list is within which.